DQM/TrackingMonitor/python/V0Monitor_cfi.py

```diff
@@ -44,5 +44,10 @@
             xmin  = cms.double( -0.5),
             xmax  = cms.double( 59.5),
       ),
+      lsPSet = cms.PSet(
+            nbins = cms.int32 ( 2000 ),
+            xmin  = cms.double(    0.),
+            xmax  = cms.double( 2000.),
```

Comment:
the code is making use of
`->getTH1()->SetCanExtend(TH1::kAllAxes);`
as in 
https://github.com/cms-sw/cmssw/pull/15447/files/d6861782b900228f801c8785a6c284fa2c6cda6b#diff-9a8083d3aef749bdf613af451ba7f731R170